src/amo/components/StaticPages/About/index.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+import Card from 'ui/components/Card';
+import Link from 'amo/components/Link';
+
+import 'amo/components/StaticPages/StaticPages.scss';
+
+
+export class AboutBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    const helpText = i18n.sprintf(i18n.gettext(`Help improve this website. It's open source, and you can file bugs and submit patches. <a href="%(url)s">GitHub</a> contains all of our current bugs, legacy bugs can still be found in Bugzilla.`),
```

Comment:
You can use newlines to keep this more readable. 

We'd ideally like to keep HTML out of the translation strings as much as possible. If the text in the link isn't translatable then you can use a placeholder for the entire link.

If the strings are translatable then we'd prefer to use a start and end tag placeholder. See https://addons-frontend.readthedocs.io/en/latest/i18n/#guidance-on-html-in-translations for examples.